fields/src/lib.rs

```diff
@@ -42,6 +42,7 @@ mod legendre;
 pub use legendre::*;
 
 mod to_field_vec;
+#[allow(unused_imports)]
```

Comment:
```suggestion
```
I'm not seeing this case appear. Can you confirm or revert?

Reply:
I confirm:
```
    Checking snarkvm-fields v0.16.18 (/path/to/snarkVM/fields)
warning: unused import: `to_field_vec::*`
  --> /path/to/snarkVM/fields/src/lib.rs:45:9
   |
45 | pub use to_field_vec::*;
   |         ^^^^^^^^^^^^^^^
   |
   = note: `#[warn(unused_imports)]` on by default
```